composer.json

```
@@ -23,6 +23,12 @@
         "ext-curl": "*",
         "oro/platform": "4.1.*"
     },
+    "require-dev": {
+        "phpunit/phpunit": "7.5.*",
+        "johnkary/phpunit-speedtrap": "3.0.*",
+        "mybuilder/phpunit-accelerator": "dev-master",
+        "symfony/phpunit-bridge": "4.4.*"
```

It makes sense to make dependencies versions synchronized with the oro/platform package. And we are not using mybuilder/phpunit-accelerator in the CI.
```suggestion
        "phpunit/phpunit": "9.1.*",
        "johnkary/phpunit-speedtrap": "3.2.*",
        "symfony/phpunit-bridge": "4.4.13"
```

---

Oops,  `phpunit-accelerator` came from `oroinc/crm-application` 😄